doc/sphinx/changes.rst

```diff
@@ -94,6 +94,106 @@ This release is the result of 384 merged PRs, closing 67 issues.
 | Nantes, January 2022,
 | Matthieu Sozeau for the Coq development team
 
+Changes in 8.15.1
```

Comment:
We usually insert this after the "Changes in 8.15.0" (I don't mind using a truly reverse chronological order, but in this case, we should update the whole changelog to be consistent).

Reply:
I hate the mixed order but fixing it is a lot of work, let's just drop consistency.

Reply:
Sorry then for introducing the mixed order (I think it's my fault). But I'd like not to be alone approving this change (to make sure that we're consistent from then on).

Reply:
>Sorry then for introducing the mixed order (I think it's my fault).

I doubt it, it's been like that for the whole changelog (eg V8 goes `8.0beta` then `8.0`)

Reply:
On second thought let's be consistent, I'll take the time to fix it all for 8.16

Reply:
Thanks!